csbindgen/src/parser.rs

```diff
@@ -12,19 +12,15 @@ enum FnItem {
 /// build a Vec of all Items, unless the Item is a Item::Mod, then append the Item contents of the vect
 /// Do this recursively.
 /// This is not memory-efficient, would work better with an iterator, but does not seem performance critical.
-fn depth_first_module_walk<'a>(ast: &'a Vec<Item>) -> Vec<&'a syn::Item> {
+fn depth_first_module_walk(ast: &Vec<Item>) -> Vec<&syn::Item> {
     let mut unwrapped_items: Vec<&syn::Item> = vec![];
     for item in ast {
-        match item {
```

Comment:
This match was weird.
Is it correct that in case it matches an `Item::Mod(...)` but without a content that it does nothing?
Current implementation reflects original code, not sure if it reflects the intention though.